lib/java/PlagiarismDetection/src/pd/SimComparer.java

```diff
@@ -89,8 +89,16 @@ private void unmarkTokens(TokenList s1Tokens, TokenList s2Tokens,
 
 	private void computeSims(TokenList s1Tokens, TokenList s2Tokens,
 			Result result) {
-		result.setSim2To1((float) s2Tokens.getMarkCount() / s2Tokens.size());
-		result.setSim1To2((float) s1Tokens.getMarkCount() / s1Tokens.size());
+		if (s1Tokens.size() == s1Tokens.getBaseCount() || s2Tokens.size() == s2Tokens.getBaseCount()) {
```

Comment:
Handles cases where there are more than one match?

Reply:
I didn't get it. Could you explain a bit furthermore?